src/services/GroupMeService.ts

```diff
@@ -37,8 +40,93 @@ export class GroupMeService implements MessagingService {
     console.log("response done");
     return right(0);
   }
+
+  /**
+   * Get list of users the bot is registered to according to the env
+   */
+  async getUsers(): Promise<Either<Error, GroupMeUser[]>> {
+    try {
+      const { members } = await this.getGroupInfo();
+      return right(members);
+    } catch (e) {
+      return left(e as Error);
+    }
+  }
+
+  /**
+   * Get group info
+   */
+  async getGroupInfo(): Promise</*Either<Error, */ GroupMeGroupInfo> /*> */ {
+    // try {
+    const { response } = await got
+      .get(`${GroupMeAPI}/groups/${this.envConfigService.GroupID}?token=${this.envConfigService.GroupMeAPIKey}`)
+      .json();
+    return response as GroupMeGroupInfo;
+    // return right(response as GroupMeGroupInfo);
+    // } catch (e) {
+    //   return left(e as Error);
+    // }
+  }
 }
 
+export type GroupMeUser = {
+  user_id: string;
+  nickname: string;
+  image_url: string;
+  id: string;
+  muted: boolean;
+  autokicked: boolean;
+  roles: string[];
+  name: string;
+};
+
+export type GroupMeGroupInfo = {
+  id: string;
+  group_id: string;
+  name: string;
+  phone_number: string;
+  type: string;
+  description: string;
+  image_url: null;
+  creator_user_id: string;
+  created_at: number;
+  updated_at: number;
+  muted_until: null; // not sure about this
```

Comment:
The response had `muted_until: null`, not sure if actual type is `string` or `number`

Reply:
Probably a nullable number